### ISSUE TITLE
Polly V8

### ISSUE THREAD

**Issue by kmcclellan** [App-vNext/Polly.Extensions.Http#36]
Is there a plan to migrate this library to work with version 8 of Polly?

**Comment by joelhulen** [comment]
@kmcclellan, no. In fact, the `Microsoft.Extensions.Http.Polly` is deprecated in favor of the new [Microsoft.Extensions.Http.Resilience](https://www.nuget.org/packages/Microsoft.Extensions.Http.Resilience) package, as [detailed here](https://learn.microsoft.com/dotnet/core/resilience/). This new package is built on top of Polly, and it takes full advantage of the new Polly v8 features.
